hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-timelineservice-documentstore/src/main/java/org/apache/hadoop/yarn/server/timelineservice/documentstore/collection/document/flowrun/FlowRunDocument.java

```diff
@@ -136,6 +140,7 @@ private TimelineMetric aggregate(TimelineMetric incomingMetric,
           .aggregate(incomingMetric, baseMetric, null);
     default:
       //NoOP
+      LOG.warn("Unknown TimelineMetricOperation.");
```

Comment:
I'd log out the operation in question.

Reply:
Thanks, I'll update. But as you can see this commit actually created 94 new findbugs issues!! So something is not quite right. I should check.